script/Arcueid/Day 8/08_00_ARC08_1_1.txt

```diff
@@ -1,22 +1,24 @@
 [sha:294fa64c2d64534c6a6c233b095275407f9c2db1]{
 -- Page 10002, Offset 19759.
 -- 兎にも角にも、まずはこの緊迫した空気を変えねばなるまい。
-Anyway, I should first try to change this tense atmosphere.
+First things first, I've got to do something about this tense atmosphere.
 }
 [sha:4eaf4c68ec1a9dd831d0c0e91c0bc264a8850ca6]{
 -- Page 10002, Offset 19760.
 -- いつもの調子では負けてしまいそうなので、ここはちょっと趣向を変えてアプローチをしてみよう。
-I feel like I'll lose if I act normal, so I'll change things up a bit and go in with a different approach.
+I feel like my usual approach isn't going to cut it here,
+ so I'll try switching things up a bit.
 }
 [sha:b211ca59827af99ac43e18b574aa7729156fc45c]{
 -- Page 35, Offset 19761.
 -- 「グモニー。いい朝だね、秋葉」
-"Good mornin'. Nice weather huh, Akiha."
+// Swap EN foreign loans for different foreign loans for effect?
```

Comment:
Mmm... Walking a thin line with this. I think this one's ok but I don't think I'd make a habit of it